examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        f"source_scripts/qaoa_params_init_{n_qubits}_qubits_{depth}_layers.npy", 
</pre><p>Why don't we use&nbsp;<code>input_data</code>&nbsp;keyword argument of Job?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
I was not aware of this. Do you mind to share a reference?

Reply:
Testing and changing to `input_data`